pkg/graph/resolvers/mto_milestone.go

```diff
@@ -101,6 +101,10 @@ func MTOMilestoneUpdate(ctx context.Context, logger *zap.Logger, principal authe
 	if err != nil {
 		return nil, fmt.Errorf("unable to update MTO Milestone. Err %w", err)
 	}
+	// Clear out the name returned from the database for convenience so we don't try to set the custom name field
```

Comment:
```suggestion
	// Since storage.MTOMilestoneGetByID will return a `Name` property when fetching milestones sourced from the common milestone library, we need to clear out that field or else storage.MTOMilestoneUpdate will attempt to update the name (which won't be allowed, since this is a Milestone sourced from the common milestone library
```